declarations.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'react-pouchdb/browser';
```

Comment:
no types available for these modules?

Reply:
Unfortunately not, there are no types libraries for them...

Reply:
then we must create local types, at least for the functions/classes that we use

see for example [what we did in the IO app](https://github.com/teamdigitale/io-app/tree/master/ts/%40types)

Reply:
@alexgpeppe remember to add these types